tvdb_v4_official.py

```diff
@@ -337,12 +333,12 @@ def get_updates(self, since: int) -> list:
 
     def get_all_tag_options(self, page=0) -> list:
         """Returns a list of tag options"""
-        url = self.url.tag_options_url(page)
+        url = self.url.tag_option_url(page)
         return self.request.make_request(url)
 
     def get_tag_option(self, id: int) -> dict:
         """Returns a tag option dictionary"""
-        url = self.url.tag_option_url()
+        url = self.url.tag_options_url()
```

Comment:
This shouldn't change the url function name, but instead it should pass the `id` arg:
```suggestion
        url = self.url.tag_option_url(id)
```